### PR TITLE
Allow to access users avatars for logged users.

### PR DESCRIPTION
**Pull request by slawekk87**
## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/Users/files/MultiImage.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	 * @param \App\Request $request
+	 *
+	 * @throws \App\Exceptions\NoPermitted
+	 *
+	 * @return bool
+	 */
+	public function getCheckPermission(\App\Request $request)
+	{
+		if (!$request->isEmpty('record')) {
+			$moduleName = $request->getModule();
```
Comment (mariuszkrzaczkowski) [inline modules/Users/files/MultiImage.php]: ?

File: modules/Users/files/MultiImage.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	 *
+	 * @throws \App\Exceptions\NoPermitted
+	 *
+	 * @return bool
+	 */
+	public function getCheckPermission(\App\Request $request)
+	{
+		if (!$request->isEmpty('record')) {
+			$fieldName = $request->getByType('field', 2);
+			if ($fieldName === 'imagename') {
```
Comment (rskrzypczak) [inline modules/Users/files/MultiImage.php]: Wrong file. imagename is not MultiImage

File: modules/Users/files/MultiImage.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+				return true;
+			} elseif (!\App\Privilege::isPermitted('Users', 'DetailView', $request->getInteger('record')) || !\App\Field::getFieldPermission('Users', $fieldName)) {
```
Comment (rskrzypczak) [inline modules/Users/files/MultiImage.php]: Why remove $moduleName?

File: modules/Users/files/MultiImage.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	 * @throws \App\Exceptions\NoPermitted
+	 *
+	 * @return bool
+	 */
+	public function getCheckPermission(\App\Request $request)
+	{
+		if (!$request->isEmpty('record')) {
+			$fieldName = $request->getByType('field', 2);
+			if ($fieldName === 'imagename') {
+				return true;
```
Comment (rskrzypczak) [inline modules/Users/files/MultiImage.php]: You can't skip permissions

**Pull request merged**